src/_includes/partials/givefeedback.njk

```diff
@@ -0,0 +1,26 @@
+<section data-pagefind-ignore class="py-100 mt-400 mb-600 {% if removeBorder %} {% else %}by-sm b-dark {% endif %}">
+  <h2 class="mt-300 mb-400">
```

Comment:
Can we change `mt-300 mb-400` to `mt-225 mb-300` please to match the new spacing classes?

Reply:
```suggestion
  <h2 class="mt-225 mb-300">
```